usr/share/rear/build/default/990_verify_rootfs.sh

```diff
@@ -128,6 +139,7 @@ if contains_visible_char "$broken_binaries" ; then
         # Show only the missing libraries to the user to not flood his screen with tons of other ldd output lines:
         PrintError "$( grep 'not found' <<<"$ldd_output" )"
     done
+    keep_build_dir
```

Comment:
It could make sense to swap this and the following line to keep related messages together.

Reply:
Done.